kubernetes/samples/scripts/create-oracle-db-service/stop-db-service.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Copyright (c) 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+# Drop the DB Service created by start-db-service.sh
+
+script="${BASH_SOURCE[0]}"
+scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
+source ${scriptDir}/../common/utility.sh
+
+function usage {
+  echo "usage: ${script} -n namespace  [-h]"
+  echo "  -n Kubernate Namespace for Oracle DB Service  (optional)"
```

Comment:
Kubernate -> Kubernetes

Reply:
Modified the string Kubernate to  Kubernetes in all scripts